ui/src/containers/ClusterInfoPage/DeleteClusterModal.tsx

```diff
@@ -66,10 +64,7 @@ export default function DeleteClusterModal({ cluster, onCancel, onDeleted }: Pro
     const message = `Cannot delete cluster. Server error occurred: "${error.message}".`;
     return (
       <InformationalModal header={`Failed to delete ${cluster.ID}!`} onAcknowledged={onCancel}>
-        <div className="flex items-center">
-          <AlertCircle size={16} className="mr-2 text-alert-600" />
-          <span className="text-lg text-alert-600">{message}</span>
-        </div>
+        <span className="pf-u-warning-color-100 pf-u-font-size-2xl">{message}</span>
```

Comment:
My first reaction is that this color needs to be "danger" (red), but then I wonder if Failed to Delete is a blocker, or just annoying but not actionable by the end user.

Do you want to suggest they post this in the #team-automation channel, just to document zombies that may be lying around?

Reply:
I was going to add the s/-warning-/-danger-/ but auto merge!